pkg/securitypolicy/README.md

```diff
@@ -0,0 +1,49 @@
+# Security Policy
```

Comment:
This might make more sense as  a package level comment so it'd show up in godoc

Reply:
Adding this to our nit list for now (to come back to this and determine the right documentation strategy), unless there are strong objections.